src/components/assignment-panel/assignment-submit-form/assignment-submit-form.tsx

```diff
@@ -47,9 +54,22 @@ export const AssignmentSubmitForm = ({ }: AssignmentSubmitFormProps) => {
                 message: 'Successfully uploaded assignment!'
             })
         } catch (e: any) {
-            snackbar.open({
+            if (e.response?.status === 409) {
+                showErrorMessage(
+                    'Push policy violation',
```

Comment:
Is this an error message that would be shown to the user? If so I don't think it's verbiage that's very good for that

Reply:
you think it's poorly worded?